buf.gen.yaml

```diff
@@ -0,0 +1,12 @@
+version: v2
+plugins:
+- remote: buf.build/protocolbuffers/go:v1.33.0
```

Comment:
Do we need to keep buf config for Go packages outside of `/apis/manager/v1beta1` directory ?

Reply:
I'm prefer global buf.gen.yaml, but there will be a lot of changes for python. Well, that's fine.